scripts/gas-refund-program/transactions-indexing/fetchRefundableTransactions.ts

```diff
@@ -167,7 +168,15 @@ export async function fetchRefundableTransactions({
     epoch,
   });
 
-  const allButV6ContractAddresses = getContractAddresses({ epoch, chainId });
+  let allButV6ContractAddresses = getContractAddresses({ epoch, chainId });
+
+  if(epoch >= GasRefundV3EpochFlip){
```

Comment:
should be other const, separate one, for the epoch when we stopped tracking augustus v5 and all networks except ethereum mainnet,
smth like
```
const EpochPIP55Adopted = 57 
```